RELEASING/README.md

```diff
@@ -487,3 +482,9 @@ click the 3-dot icon and select `Create Release`, paste the content of the ANNOU
 release notes, and publish the new release.
 
 At this point, a GitHub action will run that will check whether this release's version number is higher than the current 'latest' release. If that condition is true, this release sha will automatically be tagged as `latest` so that the most recent release can be referenced simply by using the 'latest' tag instead of looking up the version number. The existing version number tag will still exist, and can also be used for reference.
+
+### Update Superset files
+
+Now that we have a final Apache release we need to open a pull request on Superset with the changes on `CHANGELOG.md` and `UPDATING.md`.
```

Comment:
Can we make the referenced files hyperlinks (whilst insuring the backticks are preserved)?